File: main/src/main/java/net/fwitz/math/main/binary/complex/analysis/gamma/GammaPlot.java
package net.fwitz.math.main.binary.complex.analysis.gamma;

import net.fwitz.math.binary.complex.analysis.Gamma;
import net.fwitz.math.plot.binary.complex.color.DomainColoringSmooth;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class GammaPlot {
    public static void main(String[] args) {
        new ComplexFunctionPlot("gamma(z)")
                .computeFn(Gamma::gamma)
                .colorFn(new DomainColoringSmooth())
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/analysis/gamma/GammaPlot.kt
package net.fwitz.math.main.binary.complex.analysis.gamma

import net.fwitz.math.binary.complex.analysis.Gamma
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot
import net.fwitz.math.plot.binary.complex.color.DomainColoringSmooth

object GammaPlot {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("gamma(z)")
        .computeFn(Gamma::gamma)
        .colorFn(DomainColoringSmooth)
        .render()
}